models/migrations/migrations.go

```diff
@@ -76,8 +76,10 @@ var migrations = []Migration{
 
 	// v13 -> v14:v0.9.87
 	NewMigration("set comment updated with created", setCommentUpdatedWithCreated),
-
 	NewMigration("create user column diff view style", createUserColumnDiffViewStyle),
+
+	// v14 -> v15:v0.9.87
+	NewMigration("set protect branches updated with created", setProtectedBranchUpdatedWithCreated),
```

Comment:
You don't need this migration, this is only required if you want to update content, the structure gets automatically migrated.